File: Java Script - Introduction to Programming/4.2. Complex Conditions - Exam Problems/02. Trip.js
function journey([...arg]) {
  [budget, season] = [...arg]
  country = ''
  place = ''

  switch (true) {
    case budget <= 100:
      country = 'Bulgaria'
      if (season === 'summer') {
        place = 'Camp'
        budget -= budget * 0.70
      } else {
        place = 'Hotel'
        budget -= budget * 0.30
      }
      break;
    case budget > 100 && budget <= 1000:
      country = 'Balkans'
      if (season === 'summer') {
        place = 'Camp'
        budget -= budget * 0.60
      } else {
        place = 'Hotel'
        budget -= budget * 0.20
      }
      break;

    case budget > 1000:
      country = 'Europe'
      place = 'Hotel'
      budget -= budget * 0.10
      break;
  }

  console.log(`Somewhere in ${country}\n${place} - ${budget.toFixed(2)}`);
}

// journey([50, 'summer'])
// journey([1500, 'summer'])

File: Pyrhon - Introduction to Programming/4.2. Complex Conditions - Exam Problems/02. Trip.py
budget = float(input())
season = input()
country, place = '', ''

if budget <= 100:
    country = 'Bulgaria'
    if season == 'summer':
        place = 'Camp'
        budget *= 0.30
    else:
        place = 'Hotel'
        budget *= 0.70
elif 100 < budget <= 1000:
      country = 'Balkans'
      if season == 'summer':
          place = 'Camp'
          budget *= 0.40
      else:
          place = 'Hotel'
          budget *= 0.80
elif budget > 1000:
      country = 'Europe'
      place = 'Hotel'
      budget *= 0.90

print(f'Somewhere in {country}\n{place} - {budget:.2f}')


'''
June
14
#Apartment: 961.80 lv.
#Studio: 1052.80 lv
'''
